packages/api-client/index.ts

```diff
@@ -305,7 +306,21 @@ class APIClient {
         `/api/v1/courses/limited/${courseId}/${code}`,
         GetLimitedCourseResponse,
       ),
+    setCourseFeature: async (
+      courseId: number,
+      feature: string,
+      value: boolean,
+    ): Promise<void> =>
+      this.req('PUT', `/api/v1/courses/${courseId}/features`, undefined, {
```

Comment:
Maybe `PATCH` is more appropriate
![image](https://github.com/ubco-db/office-hours/assets/71345367/1518afab-4b37-4acc-97d0-f3bb4ab07bfb)

Reply:
Hmm yeah idk, I guess I originally chose PUT due to the "repeated requests have the same effect" thing. Also, I didn't want to do PATCH since the endpoint will work even if the resource doesn't exist on the server (it will create a new courseSettingsModel for the course). So, maybe POST would be the best pick?